pennylane/expval/qubit.py

```diff
@@ -156,7 +156,7 @@ class Hermitian(Expectation):
         A (array): square hermitian matrix.
         wires (Sequence[int] or int): the wire the operation acts on
     """
```

Comment:
It doesn't let me leave a comment up above, but let's make clear in the docstring that `expval.Hermitian` now accepts `N^2\times N^2` Hermitian matrix, acting on `N` wires. These docstrings act as the 'ground truth' for plugin developers to ensure they implement the correct conventions.